### PR TITLE
Updated SQL Server TDE BRS to support ready-to-run remediation feature

### PR DESCRIPTION
**Pull request by saurabhgautam-msft**

### CODE REVIEW

File: Scripts/RemediationScripts/Remediate-TransparentDataEncryptionForSQLServers.ps1 [hunk 1]
```diff
@@ -565,81 +899,97 @@ function Disable-TransparentDataEncryptionForSqlServers
...
+    }
+    else
+    {
+        Write-Host "[$($context.Account.Id)] is allowed to run this script."  -ForegroundColor $([Constants]::MessageType.Update)
+        Write-Host $([Constants]::SingleDashLine)
     }
 
-    Write-Host "*** To disable Transparent Data Encryption (TDE) for SQL Servers in a Subscription, Contributor and higher privileges on the SQL Servers are required. ***" -ForegroundColor $([Constants]::MessageType.Info)
-    Write-Host $([Constants]::DoubleDashLine)
-    Write-Host "[Step 2 of 3] Preparing to fetch all SQL Server database details..."
```
Comment (VamseeInala) [inline Scripts/RemediationScripts/Remediate-TransparentDataEncryptionForSQLServers.ps1:594]: Maintain future/present continuous tense consistently at all the places.

**Review by VamseeInala — CHANGES_REQUESTED** [review]

File: Scripts/RemediationScripts/Remediate-TransparentDataEncryptionForSQLServers.ps1 [hunk 2]
```diff
@@ -166,23 +180,38 @@ function Enable-TransparentDataEncryptionForSqlServers
...
+        [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies script is run as a subroutine of AutoRemediation Script")]
+        $AutoRemediation,
+
+        [String]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies the time of creation of file to be used for logging remediation details when AutoRemediation switch is used")]
+        $TimeStamp
     )
 
     Write-Host $([Constants]::DoubleDashLine)
-    Write-Host "[Step 1 of 4] Preparing to enable Transparent data encryption (TDE) for SQL Server database(s) in Subscription: $($SubscriptionId)"
```
[the remark this answers is not in the document]
Comment (saurabhgautam-msft) [reply Scripts/RemediationScripts/Remediate-TransparentDataEncryptionForSQLServers.ps1:173]: It should be Prepare as it is a step and should be in simple present tense. Post that action is taken thus continuous tense.

File: Scripts/RemediationScripts/Remediate-TransparentDataEncryptionForSQLServers.ps1 (hunk 1 above)
Comment (saurabhgautam-msft) [reply Scripts/RemediationScripts/Remediate-TransparentDataEncryptionForSQLServers.ps1:594]: It is maintained. Here also 'Prepare' is used.

File: Scripts/RemediationScripts/Remediate-TransparentDataEncryptionForSQLServers.ps1 (hunk 2 above)
[the remark this answers is not in the document]
Comment (saurabhgautam-msft) [reply Scripts/RemediationScripts/Remediate-TransparentDataEncryptionForSQLServers.ps1:173]: This is Red. I changed Preparing to Prepare.

**Review by VamseeInala — APPROVED** [review]

**Pull request merged**